src/pages/Recipe.test.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
 
-import Recipe from './Recipe';
+import { initialState } from 'reducers/application';
```

Comment:
Use `import { initialState } from 'reducers';` and pass that in to the mockStore instead, this gets you the whole state rather than having to assemble it as we add reducers.